code/src/java/pcgen/cdom/enumeration/SkillsOutputOrder.java

```diff
@@ -33,49 +30,9 @@
  */
 public enum SkillsOutputOrder
 {
-	NAME_ASC {
-		@Override
-		public SkillComparator getComparator(PlayerCharacter pc)
-		{
-			return new SkillComparator(pc, SkillComparator.RESORT_NAME,
-				SkillComparator.RESORT_ASCENDING);
-		}
-	},
-
-	NAME_DSC {
-		@Override
-		public SkillComparator getComparator(PlayerCharacter pc)
-		{
-			return new SkillComparator(pc, SkillComparator.RESORT_NAME,
-				SkillComparator.RESORT_DESCENDING);
-		}
-	},
-
-	TRAINED_ASC {
-		@Override
-		public SkillComparator getComparator(PlayerCharacter pc)
-		{
-			return new SkillComparator(pc, SkillComparator.RESORT_TRAINED,
-				SkillComparator.RESORT_ASCENDING);
-		}
-	},
-
-	TRAINED_DSC {
-		@Override
-		public SkillComparator getComparator(PlayerCharacter pc)
-		{
-			return new SkillComparator(pc, SkillComparator.RESORT_TRAINED,
-				SkillComparator.RESORT_DESCENDING);
-		}
-	},
-
-	MANUAL {
-		@Override
-		public SkillComparator getComparator(PlayerCharacter pc)
-		{
-			return null;
-		}
-	};
-
-	public abstract SkillComparator getComparator(PlayerCharacter pc);
+	NAME_ASC,
+	NAME_DSC,
+	TRAINED_ASC,
+	TRAINED_DSC,
+	MANUAL;
 }
```

Comment:
I would prefer the enums have the behavior, even if it means two methods.  Eliminating the switch in resortSelected would be a good thing.